packages/eslint-plugin/src/unsupported-route-components.ts

```diff
@@ -12,7 +12,7 @@ export const unsupportedRouteComponents = createRule({
     type: 'problem',
     docs: {
       description: 'Find unsupported route components',
-      recommended: 'error',
+      recommended: 'recommended',
```

Comment:
I could not find any good docs on this property and what values it should take. I need to do more reading. It was clear from TS that it wanted 1 of 3 string values. None of them were `error` though. I want to properly test this before this gets out of draft.

Reply:
The tooltip for this property is:
> The recommendation level for the rule. Used by the build tools to generate the recommended and strict configs. Exclude to not include it as a recommendation.

I think it's fine to have this be recommended. It doesn't seem to have changed anything about how the error is shown on a project when I add a `<p>` tag to the router.